projects/angular-cld/src/lib/cloudinary-image.component.ts

```diff
@@ -148,12 +148,17 @@ export class CloudinaryImage
     });
   }
 
-  placeholderHandler(options) {
+  placeholderHandler(options, image) {
     const placeholderOptions = {};
-
     Object.keys(options).forEach(name => {
-      placeholderOptions[name] = (name === 'width' && !options[name].startsWith('auto') || name === 'height') ? Math.ceil(parseInt(options[name], 10) * 0.1) : options[name];
-    });
+      if (name === 'width' && options.responsive) {
```

Comment:
@strausr , not following the logic here. 
Please simplify or add some comments to explain what you're trying to achieve. 

Question - if option.responsive is false, is it true to say that placeholderOptions === options?

Reply:
Added comment with the explanation. `Question - if option.responsive is false, is it true to say that placeholderOptions === options?` Not exactly.. we want to make use of dimension if they are provided

Reply:
I don't get this from the code.  Seems like if responsive==false, nothing happens.